dlc_developer_config.toml

```diff
@@ -21,14 +21,19 @@ datetime_tag = true
 do_build = true
 
 [test]
-# On by default
+### On by default
 sanity_tests = true
 ecs_tests = true
 eks_tests = true
 ec2_tests = true
 
-# Off by default
-sagemaker_tests = false
+### Off by default
+sagemaker_tests = true
```

Comment:
Maybe it will be more useful/clear to have sort of a switch here?
sagemaker_tests = [ "no" | "default" | "release_candidate" ]

Reply:
I like this idea a lot - let me scope this out. What about  "" | "default" | "release_candidate" - sort of the same pattern we use for partner_developer

Reply:
It's not clear what empty string means (""). I'd prefer to see something more self explaining / oblivious. 
What about "off" ? 
And later on we'll refactor *_tests from true/false to "on"/"off"

Reply:
"off" it is - i also allow "" & false - but I expect default to be "off"